deployment.rb

```diff
@@ -71,6 +71,8 @@ def load_configuration()
     'root_dir'                    => root_dir,
     'use_dynamo_tables'           => false,#[:staging, :production].include?(rack_env),
     'dynamo_table_name'           => "#{rack_env}_tables",
+    'use_dynamo_properties'       => false,
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.